### PR TITLE
Add a workshop 2022 page

### PR DESCRIPTION
**Pull request by davidscn**

### CODE REVIEW

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]:
```suggestion
If you have any questions regarding the workshop, please contact us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de).
```

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]:
```suggestion
* Early bird registration until January 17, 2022, at 23:59 CET
* Late registration until February 16, 2022
* Abstract submission until January 17, 2022, at 23:59 CET
* Recorded talks to be received until February 16, 2022
* Workshop February 21-24, 2022
```

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]: We should add SimTech here prominently. Maybe even the logo somewhere.

**Review by uekerman — COMMENTED** [review]
Just some quick comments

File: pages/community/community.md
Comment (IshaanDesai) [inline pages/community/community.md]:
```suggestion
Do you want to meet the community and learn what is new in preCICE? There is no better way than to join one of our workshops! We already had preCICE workshops in [February 2020](precice-workshop-2020.html) (Munich) and [February 2021](precice-workshop-2021.html) (online/Stuttgart), and are already planning the [next one](precice-workshop-2022.html) (online/Stuttgart). We are excited to continue to interact with all users/developers/interested-persons and enthusiasts of preCICE.
```

File: pages/community/precice-workshop-2022.md
Comment (IshaanDesai) [inline pages/community/precice-workshop-2022.md]:
```suggestion
[Registration](https://precice.org/precice-workshop-2022.html#registration) is now open. We look forward to welcoming back previous participants, and introducing more members to the preCICE community.
```

File: pages/community/precice-workshop-2022.md
Comment (IshaanDesai) [inline pages/community/precice-workshop-2022.md]:
```suggestion
Due to the uncertain times we are experiencing with the ongoing COVID-19 pandemic and the great success of the virtual workshop in February 2021, we are planning to host the workshop virtually again.
```

File: pages/community/precice-workshop-2022.md
Comment (IshaanDesai) [inline pages/community/precice-workshop-2022.md]:
```suggestion
For those who do not have the means to pay for the registration, you can write an email to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de), and we will evaluate each case individually. For selected cases we will supply a voucher to be used when registering that voids the required payment.
```

**Review by IshaanDesai — APPROVED** [review]
Some minor language corrections, rest looks good :+1:

File: pages/community/precice-workshop-2022.md
Comment (davidscn) [inline pages/community/precice-workshop-2022.md]: @IshaanDesai we can merge once we have the registration link to put it in here (and in the call for contribution section)

Comment (IshaanDesai) [reply pages/community/precice-workshop-2022.md]: Okay yes. I will update here as soon as I have the link, should be soon.

**Comment by IshaanDesai** [comment]
The registration link has been updated but is not active yet. Once the registration is activated we can merge this.

File: pages/community/community.md
Comment (uekerman) [inline pages/community/community.md]:
```suggestion
Do you want to meet the community and learn what is new in preCICE? There is no better way than to join one of our workshops! We already had preCICE workshops in [February 2020](precice-workshop-2020.html) (Munich) and [February 2021](precice-workshop-2021.html) (online/Stuttgart), and are already planning the [next one in February 2022](precice-workshop-2022.html) (online/Stuttgart). We are excited to continue to interact with all users/developers/interested-persons and enthusiasts of preCICE.
```

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]:
```suggestion
The 3rd preCICE Workshop will be held virtually through [SimTech](https://www.simtech.uni-stuttgart.de/) and the [University of Stuttgart](https://www.uni-stuttgart.de/en/), on February 21-24, 2022. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process.
```
? or a "from ... to" construction.

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]: When advertising it would be good to have sth preliminary here already. Not all participants will join all days. So, for them, it is important to see what is planned on which day.

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md:14]: Is this information needed twice? (also below)

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]:
```suggestion
For those who do not have funding to pay for the registration, you can write an email to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de), and we will evaluate each case individually. For selected cases we will supply a voucher to be used when registering that voids the required payment.
```

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]: Why not a md list here?

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]: That's the same link a third time :smile: 
I would click on all three and then be confused that they all lead to the same form.

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md:66]: Do we want to make this a must again? OK for me. But then maybe already mention in the "contribution" section.

File: pages/index.html
Comment (uekerman) [inline pages/index.html]: You could mention the date here. Can a call for contribution be "open"?

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]:
```suggestion
## Supporters
```

**Review by uekerman — APPROVED** [review]
Some minor tweaks

File: pages/community/precice-workshop-2022.md
Comment (MakisH) [reply pages/community/precice-workshop-2022.md:66]: Last year we accepted talks after the deadline and I uploaded them on YouTube just 1-2 days ahead. It was fine, but I think it would be more chaotic without a deadline a few days ahead.

If the question was about the recorded talks in general, I would vote to keep it: I really enjoyed the concept last year (good quality talks, on time, the speaker could answer questions at the same time).

File: pages/community/precice-workshop-2022.md
Comment (davidscn) [reply pages/community/precice-workshop-2022.md:14]: Are you sure you read the correct link? This one points to the registration section, not to the form.

File: pages/community/precice-workshop-2022.md
Comment (davidscn) [reply pages/community/precice-workshop-2022.md]: There is only one left now.

File: pages/community/precice-workshop-2022.md
Comment (davidscn) [reply pages/community/precice-workshop-2022.md:66]: The submission deadline or the pre-recording?

Comment (MakisH) [reply pages/community/precice-workshop-2022.md:66]: We accepted videos also after the pre-recording submission deadline.
But now that you mention this, do we want to play the "first abstract submission / now extended abstract submission deadline" game? It usually helps to remind people or to announce it twice, but then we should set the first submission deadline earlier.

File: pages/community/precice-workshop-2022.md
Comment (davidscn) [reply pages/community/precice-workshop-2022.md]: I added something. Any comment?

File: pages/community/precice-workshop-2022.md
Comment (davidscn) [reply pages/community/precice-workshop-2022.md:66]: > The submission deadline or the pre-recording?

The question referred to @uekerman's comment.

> But now that you mention this, do we want to play the "first abstract submission / now extended abstract submission deadline" game? It usually helps to remind people or to announce it twice, but then we should set the first submission deadline earlier.

Why not, the registration page is now also prepared for this. What's the problem with January 17th? We will only announce the whole workshop within this week.

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [reply pages/community/precice-workshop-2022.md:14]: Ah, ok, makes sense :+1:

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [reply pages/community/precice-workshop-2022.md:66]: The pre-recording. Do participants have to do them? I liked it, too. But we have to explain then.

Comment (davidscn) [reply pages/community/precice-workshop-2022.md:66]: I already added some explanation in the 'call for contribution' section.

Comment (MakisH) [reply pages/community/precice-workshop-2022.md:66]: > What's the problem with January 17th?

If we add an extension later for two weeks, you will only have less than a month to decide which talks to accept, announce the schedule, and speakers to prepare their presentations. It feels a bit hectic to me.

**Review by MakisH — APPROVED** [review]
Apart from the unresolved discussion regarding a potential deadline extension, every thing else looks great! :smile:

File: pages/community/precice-workshop-2022.md
Comment (IshaanDesai) [reply pages/community/precice-workshop-2022.md:66]: I think we should not plan to have an extension to  the deadline to receive the recording of the talks after 17th January. That is the deadline and we should stick to it as strictly as possible. If some participants still submit later on due to whatever reasons then we can handle the cases individually and at that moment. I have this opinion because even if we have an extended deadline we may find ourselves in a situation where we are handling some exceptional cases, and if this situation is going to occur then lets deal with it a month before the workshop.

Comment (davidscn) [reply pages/community/precice-workshop-2022.md:66]: Ah I think I misunderstood @MakisH suggestion. I agree with @IshaanDesai, let's do not plan an extension of the submission deadline. If we realize that we have too few talks/ people are late, we can still decide later on. In any case, there should be enough time for selecting talks etc.

Benjamin's point is a bit more important: I think pre-recording talks requires much more effort and having the option to do it live might be a reasonable alternative. What's your opinion @IshaanDesai ?

Comment (davidscn) [reply pages/community/precice-workshop-2022.md:66]: >  think we should not plan to have an extension to the deadline to receive the recording of the talks after 17th January. 

and I hope you mean the abstract deadline here, not the pre-recorded talks.

Comment (IshaanDesai) [reply pages/community/precice-workshop-2022.md:66]: > Benjamin's point is a bit more important: I think pre-recording talks requires much more effort and having the option to do it live might be a reasonable alternative.

I think we should not force pre-recorded talks at all. We should definitely provide the option of live talks. What we can say though is: *"From our experience we can definitely say that pre-recorded talks have worked out well in the past."*

> and I hope you mean the abstract deadline here, not the pre-recorded talks.

Yes, but more generally all deadlines..

File: pages/community/precice-workshop-2022.md
Comment (davidscn) [inline pages/community/precice-workshop-2022.md]: @IshaanDesai @MakisH @uekerman what about this version?

Comment (IshaanDesai) [reply pages/community/precice-workshop-2022.md]: This is very clear :+1: good to go from my side!

File: pages/community/precice-workshop-2022.md
Comment (MakisH) [reply pages/community/precice-workshop-2022.md:66]: > I think we should not plan to have an extension to the deadline to receive the recording of the talks after 17th January. 

I totally agree, I find the whole "extension tradition" a bit strange anyway, but this is something we have done in the past, so I wanted to know that you considered it. But let's remember to do our marketing continuously and not forget after the first batch. We could also clarify that there will be no extension, as it is quite common in conferences.

> I think we should not force pre-recorded talks at all. We should definitely provide the option of live talks. What we can say though is: "From our experience we can definitely say that pre-recorded talks have worked out well in the past."

Organizationally, this will create quite some stress/insecurity and I think the end result will be stress and mixed feelings, with some talks still going out of schedule. I think that using the power of the organizers to give a very specific guidance to people is a good thing, as it will improve the overall quality of the workshop. I also think that the way we did the online workshop last year was rather unique and we could build a good reputation by continuing in this way.

Yes, pre-recorded needs work and is stressful before the deadline, but:

- We accept talks just a few days before the conference, instead of several weeks / months before, which keeps the workload close to the time of the conference and the content still up-to-date.
- We can guarantee an almost perfect commitment to the schedule.
- The speakers will all be relaxed and participating, instead of having speakers finishing their slides the night before (very common and been there).
- Especially knowing that the talks will be on YouTube afterwards, speakers will do significant effort to prepare a very good talk.
- If we record live talks with BBB, then they will need additional manual processing to get them in a format compatible with YouTube, if we want to again make everything available online (which I think is a nice feature of this workshop).
- The interaction will be much better via the chat during the talk + Q&A afterwards.

My recommendation would be to stick to pre-recorded only. If someone really does not want to do that but wants to present, they will reach out. Then we have one special case which we can evaluate. One good reason would be that one wants to do a very interactive talk (very uncommon).

If we phrase it as "you have two options, but we suggest the high-effort option", then I expect that most people will be excited about the high-effort option in the beginning, but for pragmatic reasons they will fall back to the low-effort option near/after the deadline.

File: pages/community/precice-workshop-2022.md
Comment (MakisH) [reply pages/community/precice-workshop-2022.md]: We still need a fixed deadline for the "a few days before the workshop". If it is not on my calendar, there is no deadline, and I will postpone it until it is too late. :wink: 

I think we should still have a default with an exception, instead of offloading the decision between the two to the speakers. See previous discussion.

Comment (davidscn) [reply pages/community/precice-workshop-2022.md]: > We still need a fixed deadline for the "a few days before the workshop"

There is an exact deadline in the important dates section as described there.

Comment (MakisH) [reply pages/community/precice-workshop-2022.md]: I know, but I felt it was a bit easy to miss with this formulation. But it does not matter anyway, since one needs to look at the important dates anyway.

File: pages/community/precice-workshop-2022.md
Comment (IshaanDesai) [reply pages/community/precice-workshop-2022.md:66]: > But let's remember to do our marketing continuously and not forget after the first batch. We could also clarify that there will be no extension, as it is quite common in conferences.

Completely agree with this part. As the workshop comes closer and closer we will have a hard deadline after which we will really not accept a submission, irrespective of the any factors related to the participant.

As for forcing participants to provide pre-recorded talks, I am still not convinced. My ideas on the points raised are as follows:

> We accept talks just a few days before the conference, instead of several weeks / months before, which keeps the workload close to the time of the conference and the content still up-to-date.

I believe the content of the talk would be pretty much the same for a pre-recorded talk and a live talk. A participant would prepare the slides for a pre-recorded talk in advance, just as they would do for a live talk.

> We can guarantee an almost perfect commitment to the schedule.

I agree that pre-recorded talks give us greater control over time, but I would not think of session extension as a grave concern. We would just need the chair or another organizer to play the bad cop in the end :smiley: 

> The speakers will all be relaxed and participating, instead of having speakers finishing their slides the night before (very common and been there).

I somewhat see this to be really subjective. I would believe that a participant giving a live talk would still attend other talks and participate in discussions the day before their own talk (have been here personally).

> If we record live talks with BBB, then they will need additional manual processing to get them in a format compatible with YouTube, if we want to again make everything available online (which I think is a nice feature of this workshop).

I agree 100% here. This increases work of the chair or of some other organizer during the talks and also additional work in post processing the videos. I would vote to take it upon ourselves to do this work just to accommodate speakers who do not want to do pre-recorded talks, but surely the team should decide on this.

> The interaction will be much better via the chat during the talk + Q&A afterwards.

I honestly believe that the on-the-fly Q&A is not as effective as the talk + Q&A afterwards. I even remember instances from the 2021 workshop where speakers just replied saying that the explanation is coming up in the video soon. I even thought that having a video hampers the ability of a speaker to explain their point if they want to go back to a specific slide and really explain specific content.

Comment (MakisH) [reply pages/community/precice-workshop-2022.md:66]: > I believe the content of the talk would be pretty much the same for a pre-recorded talk and a live talk. A participant would prepare the slides for a pre-recorded talk in advance, just as they would do for a live talk.

Here I mainly wanted to differentiate between our approach and of larger conferences with pre-recorded talks. There, "presenting" two months before the Q&A was strange.

They would prepare the slides in advance, but I have often experienced last-minute live presentations, which usually do not work out. But I agree that there can also be a last-minute recording (which, however, we can reject if it is overtime).

> I somewhat see this to be really subjective. I would believe that a participant giving a live talk would still attend other talks and participate in discussions the day before their own talk (have been here personally).

Definitely would want to attend, but at least in presence conferences, people often don't join some talks or social events because of this effect. I guess in a virtual workshop this is not such a big concern.

> I honestly believe that the on-the-fly Q&A is not as effective as the talk + Q&A afterwards. I even remember instances from the 2021 workshop where speakers just replied saying that the explanation is coming up in the video soon. I even thought that having a video hampers the ability of a speaker to explain their point if they want to go back to a specific slide and really explain specific content.

But in live talks, it is very very common that there are none or only a single question, because there is no time left to discuss questions, or because there has to be all this jumping around slides to ask something from the beginning. I thought that last year we liked that the speakers could answer questions at the same time. In total, I understand that we can process more questions (with lower entry barrier) like this.

We can always upload the slides next to the video and use them in the Q&A. Also in live talks, speakers need to jump back to slides, as nobody interrupts while they are speaking.

> If we phrase it as "you have two options, but we suggest the high-effort option", then I expect that most people will be excited about the high-effort option in the beginning, but for pragmatic reasons they will fall back to the low-effort option near/after the deadline.

This point still remains open.

I don't suggest that we *force* pre-recorded talks ("oh, you want to do a live talk? Sorry, we cannot accept this."), only that we make it a special case that will be deviating from the norm ("is there a special reason you prefer this? We would like to remind you again of why we are doing this. If, for example, you want to do an interactive talk, this is perfectly fine!).

Comment (IshaanDesai) [reply pages/community/precice-workshop-2022.md:66]: Okay lets try to converge here. I understand the points @MakisH has expressed and indeed many of them are very relevant for this workshop. Overall I think the following is very much acceptable:

> I don't suggest that we force pre-recorded talks ("oh, you want to do a live talk? Sorry, we cannot accept this."), only that we make it a special case that will be deviating from the norm ("is there a special reason you prefer this? We would like to remind you again of why we are doing this. If, for example, you want to do an interactive talk, this is perfectly fine!).

Shall we go with this now?

Comment (MakisH) [reply pages/community/precice-workshop-2022.md:66]: From my side, yes, but how?

Comment (davidscn) [reply pages/community/precice-workshop-2022.md:66]: Maybe @IshaanDesai has an idea how to shape this in words?

Comment (IshaanDesai) [reply pages/community/precice-workshop-2022.md:66]: Yes, just give me some time, I'll add this.

File: pages/community/precice-workshop-2022.md
Comment (IshaanDesai) [inline pages/community/precice-workshop-2022.md]: @MakisH @DavidSCN @uekerman is this wording better now? We ask for participants to write to us in case they want to do a live presentation. The default is now pre-recorded talks.

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]:
```suggestion
Please note that this is not a classical scientific conference, but a user and developer meeting. Contributions are very welcome, if they fit the purpose, but not mandatory. We ask for contributions in the form of pre-recorded talks.
```

File: pages/community/precice-workshop-2022.md
Comment (uekerman) [inline pages/community/precice-workshop-2022.md]:
```suggestion
If you would like to submit a talk, please fill in the respective details during the registration process. We encourage all speakers to pre-record their talks (similar to the talks from [our last workshop](https://www.youtube.com/playlist?list=PLM2obW2udWdoxe1TGVuJXyIRM2JA8pOEL)) and submit them a few days before the workshop starts. Pre-recorded talks have worked out well in the past. However, for special cases, a live presentation is still possible. If you want to give a live presentation, please write to us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de) so that we can work with you for the planning of your talk. Have a look at the [important dates](https://precice.org/precice-workshop-2022.html#important-dates) below for relevant deadlines.
```

**Pull request merged**
